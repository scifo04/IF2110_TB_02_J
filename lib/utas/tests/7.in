7;
Kenapa 1 ditambah 1 itu 2?;
ya mana kutahu ehe~;
