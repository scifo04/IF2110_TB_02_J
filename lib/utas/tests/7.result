lib/utas/tests/7.in lib/utas/tests/7.out: TRUE
