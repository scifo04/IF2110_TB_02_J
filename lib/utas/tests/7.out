   | INDEX = 1
   | Joe
   | 31/10/2023 23:59:59
   | Kenapa 1 ditambah 1 itu 2?

   | INDEX = 2
   | Joe
   | 31/10/2023 23:59:59
   | ya mana kutahu ehe~

